core/server/web/middleware/url-redirects.js

```diff
@@ -93,7 +93,7 @@ _private.getBlogRedirectUrl = function getBlogRedirectUrl(options) {
  * 2. redirect to the correct admin url
```

Comment:
Also was thinking if this commend should be updated. Now that both `getBlogRedirectUrl` & `getAdminRedirectUrl` are exposed as public methods, this method is just a  wrapper around actual SSL and Admin URL redirects :thinking: